src/graph.h

```diff
@@ -37,6 +37,7 @@ class Graph : public torch::CustomClassHolder {
   int64_t GetNumRows();
   int64_t GetNumCols();
   int64_t GetNumEdges();
+  c10::intrusive_ptr<Graph> FusionSlicing(torch::Tensor seeds);
```

Comment:
Suggested renaming: `FusedBidirSlicing`. Also, can we support fused bidirectional slicing with different seeds, namely `col_index` and `row_index`?

Reply:
* Rename: Done
* Different seeds: Done